.papr.yml

```diff
@@ -17,11 +17,15 @@ env:
     GI_SCANNERFLAGS: '--warn-error'
     ASAN_OPTIONS: 'detect_leaks=0'  # Right now we're not fully clean, but this gets us use-after-free etc
     # TODO when we're doing leak checks: G_SLICE: "always-malloc"
+    TEST_TMPDIR: /srv
 
 tests:
-  - ci/ci-commitmessage-submodules.sh
-  - ci/build-check.sh
-  - ci/ci-release-build.sh
+  - docker run -e TEST_TMPDIR -e CFLAGS -e GI_SCANNERFLAGS -e ASAN_OPTIONS
+    --privileged -v $PWD:$PWD --workdir $PWD -v /srv:/srv
```

Comment:
There's nothing for this change that requires `--privileged` right?  I suspect it's just SELinux labeling that we'd hit; relabeling with `chcon -R -h -t container_file_t $PWD` and doing the same with `TEST_TMPDIR` would let us avoid being privileged, and that in turn would help guarantee these tests can be run in Kube for example.

Reply:
Good point, I added a fixup to try that out. ⬇️

Reply:
Ahh, hmm, we're still hitting

```
error: lsetxattr: Permission denied
```

when trying to commit files (with a `relabelfrom` SELinux violation).

>  I suspect it's just SELinux labeling that we'd hit; relabeling with `chcon -R -h -t container_file_t $PWD` and doing the same with `TEST_TMPDIR` would let us avoid being privileged,

But we'll still be trying to re-apply all the labels when committing to a bare repo, right? Unless we teach libglnx to only update `security.selinux` if the label isn't already set to the wanted value, though that's quite a performance hit. We could hide the behaviour behind an env var, I guess? We could also put back `--privileged` (or at least `--security-opt label:disable`) and add another testsuite to make sure the in-container path doesn't break?